macros/models/_sector/dex/enrich_dex_trades.sql

```diff
@@ -70,20 +66,16 @@ WITH base_trades as (
         {{ tokens_erc20_model }} as erc20_sold
         ON erc20_sold.contract_address = base_trades.token_sold_address
         AND erc20_sold.blockchain = base_trades.blockchain
-    LEFT JOIN
-        prices as p_bought
-        ON p_bought.minute = date_trunc('minute', base_trades.block_time)
-        AND p_bought.contract_address = base_trades.token_bought_address
-        AND p_bought.blockchain = base_trades.blockchain
-    LEFT JOIN
-        prices as p_sold
-        ON p_sold.minute = date_trunc('minute', base_trades.block_time)
-        AND p_sold.contract_address = base_trades.token_sold_address
-        AND p_sold.blockchain = base_trades.blockchain
 )
 
-select
-    *
-from
-    enrichments
+
+{{
+    add_amount_usd(
+        trades_cte = 'enrichments'
+        , prices_model = prices_model
```

Comment:
let's either:
- pass in `prices` CTE above here, similar to `enrichments`
- OR remove `prices` CTE, remove `prices_model` param in `enrich_dex_trades` macro & `dex_trades_beta` model, then just pass in `source(prices,usd)` directly here

Reply:
I pick option 2.

If option 2 failed then I will go for option 1.